packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -82,6 +82,9 @@ export const components = ({
           borderRadius: 0,
           boxShadow: "none",
 
+          '&[data-ods-type="accordion"][data-ods-variant="borderless"]': {
```

Comment:
Probably don't need the extra selector here
```suggestion
          '&[data-ods-variant="borderless"]': {
```